module2/lessons/api_consumption_in_ruby.md

```diff
@@ -178,15 +182,17 @@ end
 
 Now, we can iterate over our array and create our objects.
 
-```
+```ruby
+# lib/ghibli_films.rb
```

Comment:
Should we also mention that we need to `require './lib/film'`?

Reply:
Good point - will add